changelogs/fragments/397_vmware_content.yml

```
@@ -0,0 +1,3 @@
+breaking_changes:
```

```suggestion
minor_changes:
```

I don't think this is a breaking change since it shouldn't really _break_ existing playbooks. Going from optional to required could, but not the other way round imho. Or did I overlook something?

@goneri What do you think? I'd like to avoid a situation like #540

---

Thank you @mariolenz for raise this. Yes, I would also prefer we avoid any breaking change. Or this means we need to open up a 2.x branch.

---

@Akasurde 

Is there a reason why you think this is a breaking change? As I've said above, I don't think so. (Going from optional to required shouldn't break anything.)

If it really _does_ break existing playbooks we can't ship this in 1.6, we would need a major version 2.0. And _this_ would mean, as far as I know, that it will never end up in ansible 2.10 because this version will only ship community.vmware 1.x.

cc @goneri